server/cluster/cluster.proto

```diff
@@ -80,5 +86,9 @@ service ClusterService {
 	rpc InvalidateCache(ClusterQuery) returns (github.com.argoproj.argo_cd.v3.pkg.apis.application.v1alpha1.Cluster) {
 		option (google.api.http).post = "/api/v1/clusters/{id.value}/invalidate-cache";
 	}
-	
+
+	// ListLinks returns list of links for a cluster
+	rpc ListLinks(ClusterQuery) returns (application.LinksResponse) {
+		option (google.api.http).get = "/api/v1/clusters/{name}/links";
+	}
```

Comment:
The function does not seem to be used either. Can we remove this if not required?